ADR/0011-roles-and-permissions.md

```diff
@@ -0,0 +1,81 @@
+|               | Pipeline Results        | results.tekton.dev        | get, list                         | results, records
+|               | IntegrationTestScenario | appstudio.redhat.com      | get, list, watch                  | integrationtestscenarios
+|               | Enterprise contract     | appstudio.redhat.com      | get, list, watch                  | enterprisecontractpolicies
+|               | Release Strategy        | appstudio.redhat.com      | get, list, watch                  | releases, releasestrategies, releaseplans
+|               | Release Admission Plan  | appstudio.redhat.com      | get, list, watch                  | releaseplanadmissions
+|               | *JVM Build Service*     | jvmbuildservice.io        | get, list, watch                  | jbsconfigs, artifactbuilds
+|               | *Service Access*        | appstudio.redhat.com      | get, list, watch                  | spiaccesstokenbindings, spiaccesschecks, spiaccesstokens, spiaccesstokendataupdates
+|               | *Configs*               |                           | get, list, watch                  | configmaps
+|               | *Secrets*               |                           |                                   | secrets
+|               | Add User                |
+|               | User group (with SSO)   |
+| Maintainer    | Workspace               | Access to namespaces that backs workspace                           |
+|               | Application & Component | appstudio.redhat.com      | get, list, watch, create, update, patch | applications, components, componentdetectionqueries
+|               | Environment             | appstudio.redhat.com      | get, list, watch                        | promotionruns, snapshotenvironmentbindings, snapshots, environments
+|               | *GitOps*                | managed-gitops.redhat.com | get, list, watch                        | gitopsdeployments
+|               | PipelineRun             | tekton.dev                | create, get, list, watch                | pipelineruns
+|               | Pipeline Results        | results.tekton.dev        | get, list                               | results, records
+|               | IntegrationTestScenario | appstudio.redhat.com      | *                                       | integrationtestscenarios
+|               | Enterprise contract     | appstudio.redhat.com      | get, list, watch                        | enterprisecontractpolicies
+|               | Release Strategy        | appstudio.redhat.com      | *                                       | releases, releasestrategies, releaseplans
+|               | Release Admission Plan  | appstudio.redhat.com      | *                                       | releaseplanadmissions
+|               | *JVM Build Service*     | jvmbuildservice.io        | get, list, watch                        | jbsconfigs, artifactbuilds
```

Comment:
see https://github.com/redhat-appstudio/book/pull/42#discussion_r1067269685

Reply:
Is  this link correct?

Reply:
hmmm ... I thought I clicked and confirmed last week, but it does not look right now.

Re-searched, and found the slack thread at https://app.slack.com/client/T027F3GAJ/C02CTEB3MMF

at least today, that is the discussion in #wg-developer-stonesoup between @gorkem @ralphbean and myself on which of the jvm build service CRDs should be accessible to users.